go/libraries/doltcore/sqle/dfunctions/commit.go

```diff
@@ -15,47 +15,54 @@
 package dfunctions
 
 import (
-	"errors"
 	"fmt"
+	"strings"
 
 	"github.com/dolthub/go-mysql-server/sql"
-	"github.com/dolthub/go-mysql-server/sql/expression"
 
+	"github.com/dolthub/dolt/go/cmd/dolt/cli"
 	"github.com/dolthub/dolt/go/libraries/doltcore/doltdb"
 	"github.com/dolthub/dolt/go/libraries/doltcore/sqle"
 )
 
 const CommitFuncName = "commit"
 
 type CommitFunc struct {
-	expression.UnaryExpression
+	children []sql.Expression
 }
 
 // NewCommitFunc creates a new CommitFunc expression.
-func NewCommitFunc(e sql.Expression) sql.Expression {
-	return &CommitFunc{expression.UnaryExpression{Child: e}}
+func NewCommitFunc(args ...sql.Expression) (sql.Expression, error) {
+	return &CommitFunc{children: args}, nil
 }
 
 // Eval implements the Expression interface.
 func (cf *CommitFunc) Eval(ctx *sql.Context, row sql.Row) (interface{}, error) {
-	val, err := cf.Child.Eval(ctx, row)
+	dbName := ctx.GetCurrentDatabase()
```

Comment:
Same comment about code duplication from the other PR applies here. Do another pass on this in the near future to consolidate.

Reply:
Will do.